### PR TITLE
A Quag To The Past

### PR DESCRIPTION
**Pull request by tmagicturtle**
As per his new QC-approved Pokemon, his quotes have been updated, his custom move Murky Ambush has been changed, and his custom ability Careless has been added.

### CODE REVIEW

File: data/mods/ssb/moves.js [hunk 1]
```diff
@@ -315,65 +315,68 @@ let BattleMovedex = {
...
-	},
+    murkyambush: {
+        accuracy: true,
+        basePower: 150,
+        category: "Physical",
+        desc: "This move fails unless a foe uses a contact move on the user before the user can execute the move on the same turn. If this move is successful, the foe's move has its secondary effects suppressed and damage halved. If the user survives a hit, it attacks, and the effect ends.",
+        shortDesc: "User must be hit by a contact move before moving.",
+        id: "murkyambush",
+        name: "Murky Ambush",
+        isNonstandard: true,
```
Comment (ghost) [inline data/mods/ssb/moves.js]: should be "custom" instead of true

**Comment by ghost** [comment]
also make sure that to check the travis CI! should be tabs instead of spaces

**Comment by HoeenCoder** [comment]
I assume this is all tested?

**Comment by tmagicturtle** [comment]
Tested on my personal server, yes. Careless successfully bounces status moves and ignores boosts like Unaware; Murky Ambush ignores ground immunities and works like Shell Trap or Beak Blast.

**Comment by HoeenCoder** [comment]
Be sure to use ssb3u as a base from now on.

**Comment by HoeenCoder** [comment]
Ah dangit, didnt see the conflicts, I'm going to have to revert this. Standby.

**Comment by HoeenCoder** [comment]
Remind me to not use the revert button, fixed via force push. Rebase and re-PR to ssb3u.

**Pull request merged**